Earthfile

```diff
@@ -284,8 +284,9 @@ ruby:
   DO +BENCH --name="ruby" --lang="Ruby" --version="ruby --version" --cmd="ruby leibniz.rb"
 
 rust:
-  FROM +alpine --src="leibniz.rs"
-  RUN apk add --no-cache rust
+  FROM rust:1.64-alpine
+  DO +PREPARE_ALPINE
+  DO +ADD_FILES --src="leibniz.rs"
```

Comment:
I changed it back into the build because it is just used in the rust build, so there is no benefit of having it separated out into another function.